turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -390,6 +399,20 @@ pub fn path_to_key(path: impl AsRef<Path>) -> String {
     path.as_ref().to_string_lossy().to_string()
 }
 
+#[turbo_tasks::value(shared)]
+#[derive(Default)]
+pub struct DiskFileSystemConfig {
+    pub poll_interval: Option<Duration>,
+}
+
+#[turbo_tasks::value_impl]
+impl ValueDefault for DiskFileSystemConfig {
+    #[turbo_tasks::function]
+    fn value_default() -> Vc<Self> {
+        Self::default().cell()
+    }
+}
+
```

Comment:
Is that used?

Reply:
it is not, leftover from a previous iteration